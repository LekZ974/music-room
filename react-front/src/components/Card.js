import React from 'react';

import { Card as NBCard } from 'native-base';

const Card = props => {
  return <NBCard {...props} />;
};
export default Card;
