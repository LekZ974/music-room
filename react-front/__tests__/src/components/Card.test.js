import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../../../src/components/Card';

it('renders correctly', () => {
  const tree = renderer.create(<Card>Snapshot test with Card Component!</Card>).toJSON();

  expect(tree).toMatchSnapshot();
});
