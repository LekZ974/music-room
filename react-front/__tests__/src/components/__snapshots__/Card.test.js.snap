// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "borderBottomWidth": 0.5,
      "borderColor": "#ccc",
      "borderLeftWidth": 0.5,
      "borderRadius": 2,
      "borderRightWidth": 0.5,
      "borderTopWidth": 0.5,
      "elevation": 3,
      "flexWrap": "nowrap",
      "marginBottom": 5,
      "marginLeft": 2,
      "marginRight": 2,
      "marginTop": 5,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.1,
      "shadowRadius": 1.5,
    }
  }
>
  Snapshot test with Card Component!
</View>
`;
